pkg/rules/redigo/redigo_otel_instrumenter.go

```diff
@@ -63,6 +66,10 @@ func BuildRedigoInstrumenter() instrumenter.Instrumenter[*redigoRequest, interfa
 	builder := instrumenter.Builder[*redigoRequest, any]{}
 	getter := redigoAttrsGetter{}
 	return builder.Init().SetSpanNameExtractor(&db.DBSpanNameExtractor[*redigoRequest]{Getter: getter}).SetSpanKindExtractor(&instrumenter.AlwaysClientExtractor[*redigoRequest]{}).
+		SetInstrumentationScope(instrumentation.Scope{
+			Name:    utils.REDIGO_SCOPE_NAME,
```

Comment:
Can we use something like bellow code to get source file path of caller, I dont know the background, please ignore if it makes no sense
```go
func getScopeName() string {
	_, file, _, ok := runtime.Caller(1)
	if !ok {
		return "none"
	}
	return file
}

```

Reply:
> Can we use something like bellow code to get source file path of caller, I dont know the background, please ignore if it makes no sense
> 
> ```go
> func getScopeName() string {
> 	_, file, _, ok := runtime.Caller(1)
> 	if !ok {
> 		return "none"
> 	}
> 	return file
> }
> ```

The name is not the path of caller, It is the relative path of the `setup`. Will your `getScopeName` method return a absolute path?